app/index.js

```diff
@@ -358,39 +358,32 @@ app.on('ready', () => {
   process.on('window-alert',
     (webContents, extraData, title, message, defaultPromptText,
```

Comment:
i think if you replaced braveExtensionId in `title` with 'Brave', that would resolve https://github.com/brave/browser-laptop/issues/7280